modules/azure/loadbalancer.go

```diff
@@ -0,0 +1,217 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-09-01/network"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// IPType enumerator for IP Types, Public or Private.
+type IPType int
+
+// IPType values listed using iota
+const (
+	PublicIP IPType = iota
+	PrivateIP
```

Comment:
NIT: You might want to rename `IPType` and the values to `LoadBalancerIPType`, and/or define it in a separate `enums.go` file, since the namespace is shared across the whole `azure` package. I suspect this would be a common enum to use across the module.

Reply:
Moved